cmake/Install.cmake

```diff
@@ -456,23 +456,23 @@ if(WIN32)
   install(FILES "${PROJECT_SOURCE_DIR}/bin/IDFVersionUpdater/Run-Win/IDFVersionUpdater Libs/Shellx64.dll"
           DESTINATION "PreProcess/IDFVersionUpdater/IDFVersionUpdater Libs/")
   install(FILES "${PROJECT_SOURCE_DIR}/bin/IDFVersionUpdater/Run-Win/icudt65.dll"
-          DESTINATION "PreProcess/IDFVersionUpdater/icudt65.dll")
+          DESTINATION "PreProcess/IDFVersionUpdater/")
   install(FILES "${PROJECT_SOURCE_DIR}/bin/IDFVersionUpdater/Run-Win/icuin65.dll"
-          DESTINATION "PreProcess/IDFVersionUpdater/icuin65.dll")
+          DESTINATION "PreProcess/IDFVersionUpdater/")
   install(FILES "${PROJECT_SOURCE_DIR}/bin/IDFVersionUpdater/Run-Win/icuuc65.dll"
-          DESTINATION "PreProcess/IDFVersionUpdater/icuuc65.dll")
+          DESTINATION "PreProcess/IDFVersionUpdater/")
   install(FILES "${PROJECT_SOURCE_DIR}/bin/IDFVersionUpdater/Run-Win/msvcp120.dll"
-          DESTINATION "PreProcess/IDFVersionUpdater/msvcp120.dll")
+          DESTINATION "PreProcess/IDFVersionUpdater/")
   install(FILES "${PROJECT_SOURCE_DIR}/bin/IDFVersionUpdater/Run-Win/msvcp140.dll"
-          DESTINATION "PreProcess/IDFVersionUpdater/msvcp140.dll")
+          DESTINATION "PreProcess/IDFVersionUpdater/")
   install(FILES "${PROJECT_SOURCE_DIR}/bin/IDFVersionUpdater/Run-Win/msvcr120.dll"
-          DESTINATION "PreProcess/IDFVersionUpdater/msvcr120.dll")
+          DESTINATION "PreProcess/IDFVersionUpdater/")
   install(FILES "${PROJECT_SOURCE_DIR}/bin/IDFVersionUpdater/Run-Win/vccorlib140.dll"
-          DESTINATION "PreProcess/IDFVersionUpdater/vccorlib140.dll")
+          DESTINATION "PreProcess/IDFVersionUpdater/")
   install(FILES "${PROJECT_SOURCE_DIR}/bin/IDFVersionUpdater/Run-Win/vcruntime140.dll"
-          DESTINATION "PreProcess/IDFVersionUpdater/vcruntime140.dll")
+          DESTINATION "PreProcess/IDFVersionUpdater/")
   install(FILES "${PROJECT_SOURCE_DIR}/bin/IDFVersionUpdater/Run-Win/XojoGUIFramework64.dll"
-          DESTINATION "PreProcess/IDFVersionUpdater/XojoGUIFramework64.dll")
+          DESTINATION "PreProcess/IDFVersionUpdater/")
```

Comment:
Got it, so [DESTINATION](https://cmake.org/cmake/help/latest/command/install.html#introduction) should be a directory, not the target filename. :+1: